### PR TITLE
new chang in cards

### PR DESCRIPTION
**Pull request by Royamosavipour**
Hey
Respectfully, new changes have been made, please review and advise if possible
Thanks

### CODE REVIEW

File: app/src/components/Card/Sections/Attachments.tsx [hunk 1]
```diff
@@ -1,21 +1,94 @@
...
+  RiArrowLeftSLine,
+  RiArrowRightSLine,
+  Minus,
+  Plus,
+  DownloadIcon,
+} from '../../Project/Content/IconsCard'
+import { useDisclosure } from '@mantine/hooks'
 
-function FileTile({ file }: { file: IFile }) {
-  switch (file.type) {
```
Comment (eugeek) [inline app/src/components/Card/Sections/Attachments.tsx:6]: Actually this component won't use only for images, and I don't get it why you change this code 🙃

File: app/package.json [hunk 2]
```diff
@@ -52,6 +52,7 @@
     "react-firebase-hooks": "^5.1.1",
     "react-icons": "^4.12.0",
     "react-router-dom": "^6.16.0",
+    "swiper": "^11.1.3",
```
Comment (eugeek) [inline app/package.json:55]: Do we actually need in this package? I didn't see the usage of this one

**Comment by eugeek** [comment]
LGTM 👍

**Comment by Royamosavipour** [comment]
The changes were made successfully

**Pull request merged**